### DISCUSSION TITLE
External developers cannot see all issues on a project

### DISCUSSION THREAD

**Discussion by nexusz** [community/community#128104] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello,

I have several external developers that I added to my private repository
Associated to these repositories there are projects with issues.
There are, however, issues that can be seen by me, but not by hte external developer.

How can this be? If the developer has access to the repository and project, he/she should see the issues in the project no mather what.
What am I doing wrong

Kind regards,

**Comment by mdekaa** [comment]
ensure that the external developers have the correct level of access to the repository. 

Go to your repository on GitHub.
Click on "Settings" > "Manage access" (or "Collaborators and teams").
Verify that the external developers are listed as collaborators or members of a team with the appropriate permissions.

Another problem might be its related to project board permission , the board itself might have some restrictive permission. So better to check those too.
Navigate to the project board linked with the issues.
Click on "Menu" > "Project settings" > "Access."
Ensure the project board permissions allow external developers to view all issues.

**Reply by ebndev** [reply]
@mdekaa thanks for taking the time to answer!

@nexusz - let us know if this works 😄
